packages/webapp/src/components/Modals/QuickAssignModal/styles.module.scss

```diff
@@ -29,3 +29,6 @@
   width: 96px;
 }
 
+.warning {
+  color: var(--brown700);
```

Comment:
Couldn't find the UI specs on the Jira ticket -- we do have a colors.scss file that has a Litefarm standard warning color. Ignore this if this color was specifically asked for.

Reply:
I checked figma, and this color was required.